### PR TITLE
prevent app from crashing on "share data" click 

### LINKED ISSUE #4970: Status-Desktop is not usable after creating account if you click on "Share Data with Status"
Issue found and reported by @churik 

# Status-Desktop is not usable after creating account if you click on "Share Data with Status"
### Steps
- Open StatusIm
- Create account
- Tap on "Share data". Popup with confirmation should appear.
- Tap on "Yes" or "Cancel"

### Expected result
can use Status

### Actual result
merged status

* OS: Mac OSx High Sierra
* Log: https://drive.google.com/file/d/1q5-9vglUXIhUt9znUsIy744QYju6UQRJ/view?usp=sharing
* Note: confirmation popup is missing in `desktop` branch, so the issue is presented only in particular PR. Only workaround is click on "Don't share".

### PR DESCRIPTION
**Pull request by rcullito**
Fixes #4970 

Simple fix. Assuming for now we just want the app to "not crash" for desktop P0. Instabug, etc... to be revisited later.

### CODE REVIEW

File: src/status_im/ui/screens/usage_data/views.cljs [hunk 1]
```diff
@@ -28,12 +28,9 @@
...
                                  :uppercase?        false
-                                 :on-press          #(utils/show-confirmation {:ios-confirm-style "default"}
-                                                                              (i18n/label :t/confirmation-title)
-                                                                              (i18n/label :t/confirmation-text)
-                                                                              (i18n/label :t/confirmation-action)
-                                                                              (fn [] (re-frame/dispatch [:help-improve-handler true next]))
-                                                                              nil)
+                                 ;; this is a desktop hack. we are currently not collecting data usage
+                                 ;; on behalf of the user
+                                 :on-press          #(re-frame/dispatch [:help-improve-handler false next])
```
Comment (maxhora) [inline src/status_im/ui/screens/usage_data/views.cljs]: Should we outline that we do this only for `desktop` platform?

Comment (vkjr) [reply src/status_im/ui/screens/usage_data/views.cljs]: Agreee. Every place where we modify few lines better to be enclosed with `(if platform/desktop? ..). That will make merging with `defelop` easier.

Comment (rcullito) [reply src/status_im/ui/screens/usage_data/views.cljs]: right. agreed with you both. thanks for the feedback. will make the follow up change.

**Review by vitvly — APPROVED** [review]

**Comment by rcullito** [comment]
@churik anything else I should do for this to move into Test? Looks like the status-github-bot moved it back  :eye:

**Review by vkjr — APPROVED** [review]

**Review by goranjovic — APPROVED** [review]

**Comment by churik** [comment]
Tested: 
- sign up flow
- "Share Data" after Sign Up
- "Don't share data" after Sign Up

**Pull request merged**